### PR TITLE
feat: add aks byok(SSE+CMK) support

### PR DESCRIPTION
**Pull request by andyzhangx**
aks-preview:
* Update to use 2020-01-01 api-version
* Support cluster creation with server side encryption using customer managed key

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by andyzhangx** [comment]
also @sauryadas 
pls take a review first, I did not add generated *.whl file to storage account since there are two AKS PRs ahead, there could be version conflict.

I have tried following command works well with new api-version`v2020-01-01`:
```
az aks create -g $RESOURCE_GROUP_NAME -n $CLUSTER_NAME --node-count 1 --generate-ssh-keys  --kubernetes-version 1.17.0 --node-osdisk-diskencryptionset-id /subscriptions/{subsid}/resourceGroups/andy-diskencryptionset/providers/Microsoft.Compute/diskEncryptionSets/test-des
```

**Comment by andyzhangx** [comment]
cc @zikalino

**Comment by zqingqing1** [comment]
FYI, Bin Xia has another PR which also used 0.4.25 for release. You may change the version if his PR merged before yours.

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -210,7 +213,8 @@
           text: az aks create -g MyResourceGroup -n MyManagedCluster --load-balancer-sku basic --vm-set-type AvailabilitySet
         - name: Create a kubernetes cluster with authorized apiserver IP ranges.
           text: az aks create -g MyResourceGroup -n MyManagedCluster --api-server-authorized-ip-ranges 193.168.1.0/24,194.168.1.0/24,195.168.1.0
-
+        - name: Create a Kubernetes cluster with server side encryption using your owned key.
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/_help.py]: NIT: please lowercase "Kubernetes"

**Comment by andyzhangx** [comment]
don't merge this PR since this PR is not completed, will wait for https://github.com/Azure/azure-cli-extensions/pull/1111 merge first

File: src/index.json [hunk 2]
```diff
@@ -88,9 +88,10 @@
                     "metadata_version": "2.0",
                     "name": "aks-preview",
                     "summary": "Provides a preview for upcoming AKS features",
-                    "version": "0.4.25"
+                    "version": "0.4.26"
                 },
-                "sha256Digest": "58f04bc4417783b102ae83481338e9a16567a9183a021dff5a1f6c444b591f80"
+                "sha256Digest": "572dfc10ff045b00a0e65528afd415b14dfaade3b8d51054b7fc45c13e81f61c"
+                    "version": "0.4.25"
```
Comment (andyzhangx) [inline src/index.json]: why there is ` "version": "0.4.25"`? wrong push?

Comment (zikalino) [reply src/index.json]: oh, wait, seems like we are doing sth at the same time
i was resolving conflicts

Comment (zikalino) [reply src/index.json]: ok, could you push proper version? i will merge then

Comment (andyzhangx) [reply src/index.json]: ok, I will leave this PR to you, thanks.

Comment (andyzhangx) [reply src/index.json]: `https://azurecliaks.blob.core.windows.net/azure-cli-extension/aks_preview-0.4.26-py2.py3-none-any.whl` is the correct file now, just remove `"version": "0.4.25"` should be ok

**Pull request merged**